### PR TITLE
10 sort reviews

### PR DESCRIPTION
**Pull request by Chris-Duignan**
- add expandable sort form for setting queries
- update fetch reviews function to accept query params
- update url with search query
- add error handling
- add basic styling

(CSS needs refactoring across the app)

### CODE REVIEW

File: src/components/SortForm.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setSearchParams(searchTerm);
+  };
+
+  return (
+    <form onSubmit={handleSubmit} className="sortForm">
+      <label>
+        Sort By:
+        <select name="sort_by" id="sort_by" onChange={handleChange}>
```
Comment (Augs0) [inline src/components/SortForm.jsx]: This select is currently uncontrolled. Please make sure that the select's value is tied to your state object

**Review by Augs0 — CHANGES_REQUESTED** [review]
This is some really wonderful work so far, Chris! A few things to look at before you move on:

- your select dropdown is currently uncontrolled, so we just need to make sure the value for the select (not the options) is linked to our state
- Your categories dropdown menu doesn't close on click so it kinda gets in the way of your content which is not a good user experience
- Please add the ability to filter in the category pages as well as all reviews. If I go and look at a select group of reviews, I would still like to sort and order them :) 

Again, excellent work so far - keep it up!

File: src/components/SortForm.jsx (hunk 1 above)
Comment (Chris-Duignan) [reply src/components/SortForm.jsx]: No problem, I'll start working on them. Should I do another pull request on slack once I've done that?

**Review by Augs0 — APPROVED** [review]
Smashing, Chris! You've handled everything I suggested :D 🎉

**Pull request merged**